subgraph.template.yaml

```
@@ -21,52 +30,63 @@ dataSources:
         - event: Transfer(indexed address,indexed address,indexed uint256)
           handler: handleTransfer
       file: ./src/erc721/index.ts
+    {{/erc721}}
 
-  - name: ERC7432
+{{#erc1155}}
```

Decide if you are gonna leave tabs here. In `erc721` you left
```suggestion
    {{#erc1155}}
```